sandbox/experimental/data-packages/dwc-dp/0.1/table-schemas/genetic-analysis.json

```diff
@@ -0,0 +1,128 @@
+{
+  "name": "genetic-analysis",
+  "identifier": "http://rs.gbif.org/data-packages/dwc-dp/genetic-analysis",
+  "url": "http://rs.gbif.org/sandbox/experimental/data-packages/dwc-dp/0.1/table-schemas/genetic-analysis.json",
+  "title": "GeneticAnalysis",
+  "description": "A link between a DNA sequence and the event and material from which it was derived, using a specified protocol",
+  "fields": [
+    {
+      "name": "geneticAnalysisID",
```

Comment:
Do you normally have such an ID? If not, a compound PK across eventID, geneticProtocolID, geneticSequenceID, materialID could be used.

Reply:
We don´t. But we were trying to also accommodate targeted assays (pcr) where the sequence does not exist. Then we need to be able to link directly from `Identification` through `identificationBasedOnGeneticAnalysisID`.

Reply:
Ah of course - thanks. Wasn't thinking this through far enough.

Reply:
And without the unique ID, it there wouldn't be a consistent way to link Assertions to it.